ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/common/worldview/accumulator/DiffBasedWorldStateUpdateAccumulator.java

```diff
@@ -504,6 +504,8 @@ public void commit() {
                 tracked.setStorageWasCleared(false); // storage already cleared for this transaction
               }
             });
+    getUpdatedAccounts().clear();
+    getDeletedAccounts().clear();
```

Comment:
I suggest to do this clearing in a separate PR as it is not related to parallelization

Reply:
yes will remove it . moved to https://github.com/hyperledger/besu/pull/7974